docs/dietpi_tools.md

```diff
@@ -192,11 +192,26 @@ When you execute the certificate installation it also installs it for your selec
     To be accessible from the internet, typically your router needs a port forwarding configuration to route incoming HTTP and HTTPS accesses to your DietPi system.  
     Although you only need a HTTPS protocol forwarding (typically port 433), you also need to forward the HTTP protocol (typically port 80) to your DietPi system, otherwise the certification renewal procedure will fail (due to the fact that the certification renewal procedure takes place several months later you may have forgotten this issue).
 
-### DietPi NordVPN
+### DietPi VPN
 
-Run `dietpi-nordvpn`.
+DietPi-VPN is a combination of OpenVPN installation and DietPi front end GUI. Allowing all VPN users to quickly and easily connect to any NordVPN, ProtonVPN, or any other server that uses OpenVPN in TCP or UDP, using only open source software.
 
-![DietPi-NordVPN screenshot](assets/images/dietpi-nordvpn.jpg){: width="642" height="207" loading="lazy"}
+![DietPi-VPN screenshot](assets/images/dietpi-vpn.jpg){: width="642" height="312" loading="lazy"}
+=== "Requires VPN Subscription"
```

Comment:
1.) Empty line necessary between lines 199 and 200, otherwise dietpi-vpn.jpg is not displayed.

2.) I hope that the username is no real user name from you. Otherwise I propose to change it.

3.) The resolution of the screenshot seems to be possible to be increased a bit. It seems that there are dithering effects visible.

4.) I propose not to use blue borders left and right beside the grey main dialog box. See other images in the page https://dietpi.com/docs/dietpi_tools.

Reply:
Don't worry, the username's just a keyboard smash.  It's kind of similar to the ProtonVPN OpenVPN usernames, though.